deploy/helm/pumrpmicro/values.yaml

```diff
@@ -3,26 +3,24 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 apiversion:
-  deployment: apps/v1beta1 # for versions 1.8+ use apps/v1beta2
+  deployment: apps/v1 # for versions 1.8 use apps/v1beta2
 image:
```

Comment:
We could also add a [ConfigMap](https://kubernetes.io/blog/2016/04/configuration-management-with-containers/) on the helm chart to be able to customize the environment variables from the images to configure the Mongo connection, instead of doing it at the docker image creation level. I'm doing something similar on my end, in case you are interested we can work on it on a separate PR.

Reply:
I've already done this via environment variables! Take a look at the order and catalog readme helm command.

Reply:
this one? helm install ./deploy/helm/pumrpmicro --name=order --set image.name=pumrp-order,image.repository=microsoft